analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -84,7 +84,7 @@ abstract class AbstractIntegrationSpec : StringSpec() {
         // Do not use the usual simple class name as the suffix here to shorten the path which otherwise gets too long
```

Comment:
Commit message nit: A comma is missing before the "and" after "configured" . Alternatively, the sentence could be split into two as it is rather long.